bin.src/imsim.py

```diff
@@ -37,8 +38,18 @@
                     help='integer used to seed random number generator')
 parser.add_argument('--processes', type=int, default=1,
                     help='number of processes to use in multiprocessing mode')
+parser.add_argument('--image_dir_path', type=str, default=None,
+                    help="search path for FITS postage stamp images."
+                    "This will be prepended to any existing IMAGE_DIR_PATH"
+                    "environment variable, for which $CWD is included by"
+                    "default.")
 args = parser.parse_args()
 
+if args.image_dir_path is not None:
+    os.environ['IMAGE_DIR_PATH'] \
```

Comment:
IMAGE_DIR_PATH seems a bit generic. Probably we will never have a conflict but to help people who are reseting things etc and looking at printenv maybe we could make it IMSIM_IMAGE_DIR_PATH or FITS_IMAGE_DIR_PATH?

Reply:
I worried about this too.  `IMSIM_IMAGE_PATH` seems sufficiently non-generic.